app/lib/tufts/importer.rb

```diff
@@ -79,9 +79,10 @@ def records
     #
     # @return [Enumerable<Error>]
     # @see #errors
-    def validate!
+    def validate!(import_type = 'xml')
+      @import_type = import_type
       check_for_well_formed_xml(@file.read)
-      validate_filenames
+      validate_filenames unless @import_type == 'metadata'
```

Comment:
:+1: I wish we had a better way to do this, but I like this solution for its simplicity.